pkg/server/plugin/nodeattestor/x509pop/x509pop.go

```diff
@@ -84,10 +96,22 @@ func buildConfig(coreConfig catalog.CoreConfig, hclText string, status *pluginco
 		pathTemplate = tmpl
 	}
 
+	var SPIFFEPrefix string
```

Comment:
local variable naming conventions are `lowerCamel`
```suggestion
	var spiffePrefix string
```